### PR TITLE
Add CONTRIBUTING

### PR DESCRIPTION
**Pull request by natct10**
This PR is a contributing guideline general proposition, plus some tips for organizing a collective workflow. 
Please let me know if any clarification is need!
Cheers

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Contributing to Contributivity
```
Comment (RomainGoussault) [inline CONTRIBUTING.md]: nice start :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+conda create --name NAME_OF_YOUR_VENV
+
+# Activate your new virtual environment
+conda activate NAME_OF_YOUR_VENV
+
+# Stop your virtual environment
+deactivate
+```
+
+Once inside your new virtual environment, you can install the project dependencies with the command `pip3 install -r requirements.txt`. What is installed inside a virtual environment is separated from your general Python setup.
```
Comment (RomainGoussault) [inline CONTRIBUTING.md]: For conda: `conda install --file requirements.txt`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+#### Black formatter
+
+This tool might seem a little bit radical, but it is based on PEPs and offers several possibilies of customization. It will help you learn and improve your code
+
+You don't have to take all the suggested modifications (with the help of `git diff`) but it is a good reference based on PEP rules that will ensure **validity**, **maintainability** and **readability** of the code: [Package Repository](https://github.com/psf/black)
+
+##### Installation
+
+```sh
+pip3 install black
```
Comment (RomainGoussault) [inline CONTRIBUTING.md]: `conda install -c conda-forge black
`

File: CONTRIBUTING.md (hunk 2 above)
Comment (natct10) [reply CONTRIBUTING.md]: Oh yes, thank you @RomainGoussault

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+##### Usage
+
+```sh
+# Inspect a file
+black FILE.py
+# Inspect files in the current folder
+black .
+```
+
+#### Flake8 linter
```
Comment (RomainGoussault) [inline CONTRIBUTING.md]: What is the difference linter vs formatter ?

Comment (natct10) [reply CONTRIBUTING.md]: I naively would say that a linter deals with the quality of the code while a formatter focuses on the structure/format of the doc, it doesn't "evaluate"

File: CONTRIBUTING.md (hunk 3 above)
Comment (natct10) [reply CONTRIBUTING.md]: Thx!

**Pull request merged**